GDevelop.js/__tests__/Core.js

```diff
@@ -1108,6 +1108,29 @@ describe('libGD.js', function () {
       expect(allResources.get(0)).toBe('Used');
       project.delete();
     });
+
+    it('should not remove loading screen image when removing useless resources', function () {
+      var project = gd.ProjectHelper.createNewGDJSProject();
+      var resource1 = new gd.ImageResource();
```

Comment:
nitpicking: you could as well use `const` here (by default in JS, I try to use `const` for all variables declaration, unless it will change later) (this avoided a few errors in the past, where you try to re-assign a variable by mistake).